business/dashboards.go

```diff
@@ -62,7 +62,11 @@ func (in *DashboardsService) GetDashboard(params prometheus.CustomMetricsQuery,
 			defer wg.Done()
 			filledCharts[idx] = models.ConvertChart(chart)
 			if chart.DataType == kubernetes.Raw {
-				filledCharts[idx].Metric = in.prom.FetchRange(chart.MetricName, labels, grouping, &params.BaseMetricsQuery)
+				aggOp := params.RawDataAggregator
```

Comment:
@jotak Not a blocker but you could carry through "aggregator" terminology.  I see "aggOp", "op" and "AggregationOperator" in here.